tests/src/test/javascript/msltests.html

```diff
@@ -89,6 +89,7 @@
 <script type="text/javascript" src="../../../../core/src/main/javascript/entityauth/PresharedProfileAuthenticationData.js"></script>
 <script type="text/javascript" src="../../../../core/src/main/javascript/entityauth/ProvisionedAuthenticationData.js"></script>
 <script type="text/javascript" src="../../../../core/src/main/javascript/entityauth/RsaAuthenticationData.js"></script>
+<script type="text/javascript" src="../../../../core/src/main/javascript/entityauth/EccAuthenticationData.js"></script>
```

Comment:
Please alphabetize as long as there are no dependencies that require a different file order.